### DISCUSSION TITLE
#url ||and|| Local Url

### DISCUSSION THREAD

**Discussion by Khampol** [open-webui/open-webui#7502] [Q&A]
Hi,
 I am looking for the answer and do not find something so ask here : 

 I try ex. (ex. url)
#https://www.treeshark.com/treeblog/
This do not work 

I also have some html files and put it in local and try 
#http://192.168.11.5
This also do not work, "Url invalid...."

in both I try enable and disable "web serch" : same result... :-/
Did I miss something ?

**Comment by plachance** [comment]
Was having the same question on v0.5.11.

Since I don't know which version, you have to start the prompt with #the_url then click on the autocomplete tooltip/drop-down that appears above the chat box. The URL is then loaded as if it's a document from the workspace.

You can add a document from the workspace at the end of the prompt, but for URLs, it must be at the start of the prompt and the chat box must be empty for some reason. It's not very user-friendly.

Reference: https://docs.openwebui.com/features/rag#local-and-remote-rag-integration
